src/MockBot.Api/Program.cs

```diff
@@ -23,7 +24,8 @@ public static void Main(string[] args)
 
             // Add services to the container.
             var dmrSettings = configuration.GetSection("DmrServiceSettings").Get<DmrServiceSettings>();
-            services.AddDmrService(dmrSettings);
+            var centOpsSettings = configuration.GetSection("DmrServiceSettings").Get<CentOpsServiceSettings>();
```

Comment:
This looks like `dmrSettings` and `centOpsSettings` are reading the same section. Is this right?

Reply:
Perhaps consider adding a new section for the centops settings.